person_class.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id

```

Comment:
- [OPTIONAL] For better code organization, it will be nice to move these variable declarations to the top of your class.